src/components/views/read/PlayerView.tsx

```diff
@@ -84,17 +76,45 @@ const PlayerView: FC = () => {
     'Template',
     'Review',
   ];
-  const [cvValues, setCvValues] = useState<Record<string, any>>({});
-
-  const handleModify = (category: string, modifiedValues: any): void => {
-    setCvValues((prevValues) => ({
-      ...prevValues,
-      [category]: modifiedValues,
-    }));
+  // const [cvValues, setCvValues] = useState<Record<string, any>>({});
+
+  // const handleModify = (category: string, modifiedValues: any): void => {
+  //   setCvValues((prevValues) => ({
+  //     ...prevValues,
+  //     [category]: modifiedValues,
+  //   }));
+  // };
+
+  const [cvValues, setCvValues] = useState<CVInfoObj>({
+    personalInfo: {
+      firstName: '',
+      lastName: '',
+      birthDate: undefined,
+      gender: '',
+      emailAddress: '',
+      phoneNum: '',
+      address: '',
+      profileLinks: '',
+      personalLink: '',
+      personalPic: '',
+      // ... other properties with initial values
+    },
+    educationInfo: [],
+    workInfo: [],
+    skillsInfo: [],
+    portfolioInfo: [],
+    motivationInfo: {
+      motivationLetter: '',
+    },
+    referencesInfo: [],
+  });
+
+  const handleCvValuesChange = (newCvValues: CVInfoObj): void => {
+    setCvValues(newCvValues);
   };
 
   return (
-    <div
+    <Box
       style={{
         fontFamily: 'Roboto',
```

Comment:
What do you mean with that, I didn't get it.

Reply:
By default, the MUI theme uses these values: https://mui.com/material-ui/customization/default-theme/ if you look at the typography section, you have `fontFamily: "Roboto", ...` so roboto is used as the default font. You can therefore remove it from the style prop here, as it can be redundant.